src/common/app-paths/app-path-injection-token.ts

```diff
@@ -2,11 +2,6 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import { getInjectionToken } from "@ogre-tools/injectable";
 import type { PathName } from "./app-path-names";
 
 export type AppPaths = Record<PathName, string>;
-
-export const appPathsInjectionToken = getInjectionToken<AppPaths>({ id: "app-paths-token" });
```

Comment:
Very interesting that this isn't used in renderer any more. I wonder what made it obsolete.

Reply:
Because we have a single State implementation that is in common and used to create all the dependent injectables.